src/ufs.rs

```diff
@@ -408,7 +408,15 @@ impl Filesystem for Ufs {
 					let len = ino.size as usize;
 					Ok(link[0..len].to_vec())
 				}
-				_ => Err(IoError::new(ErrorKind::Unsupported, "TODO: long links")),
+				InodeData::Blocks { .. } => {
+					// TODO: I'm not sure if symlinks can be longer than a fragment
+					assert_eq!(ino.blocks, 1);
```

Comment:
There's an easy way to tell; create a symlink of maximum length in the golden image.

Reply:
1024 seems to be max, at least for 32K/4K.